### ISSUE TITLE
Default Host/Port

### ISSUE THREAD

**Issue by Jack-Chronicle** [notjedi/gotem#1]
Might be on the roadmap already, but wanted to ask for an option to have a default Host, Port, and other flags as an option

**Comment by notjedi** [comment]
yes, that is in the roadmap but it might take a while for me to actually get to that cause i'm caught up in work these days.

**Comment by Jack-Chronicle** [comment]
I'll just set up an alias as a temporary measure till then in that case
